lib/tumblr.js

```diff
@@ -101,9 +121,7 @@ TumblrClient.prototype = {
 
   // Posts
 
-  edit: function (blogName, options, callback) {
-    this._post(blogPath(blogName, '/post/edit'), options, callback);
-  },
+  edit: calls.postEditing(blogName, options, callback),
```

Comment:
for post editing, you don't have multiple routes to make the `calls.postEditing` stuff you added necessary the way it is for creating a new post. as it is now, all the stuff being passed into `calls.postEditing` is undefined (and ultimately gets ignored anyway), so you want this instead:
```js
  edit: function(blogName, options, callback) { 
    options = setDefaultPostOptions(options);
    this._post(blogPath(blogName, '/post/edit'), options, callback);
  },
```